kafka/channel/config/500-kafka-ch-dispatcher.yaml

```diff
@@ -0,0 +1,83 @@
+# Copyright 2018 The Knative Authors
```

Comment:
the file could be named `500-dispatcher.yaml` to match the others?

also, we had that name for this file before... see:
https://github.com/knative/eventing-contrib/blob/release-0.12/kafka/channel/config/500-dispatcher.yaml

related PR from FEB: #922

Reply:
+1

Reply:
Done